packages/fondue/src/components/FilterableMultiSelect/FilterableMultiSelect.tsx

```diff
@@ -55,7 +55,7 @@ export type FilterableMultiSelectProps = {
     emptyFilteredResultsLabel?: string;
 };
 
-export type Item = {
+type FilterableMultiSelectItem = {
```

Comment:
<img width="737" alt="Screenshot 2024-03-26 at 09 52 07" src="https://github.com/Frontify/fondue/assets/13820687/cc9d268f-36bd-4383-8bfd-6b02778a5d05">
May this cause a naming conflict?